fbw-common/src/wasm/systems/systems/src/hydraulic/linear_actuator.rs

```diff
@@ -807,6 +806,22 @@ impl CoreHydraulicForce {
             .set_max(self.max_control_force.output().get::<newton>());
     }
 
+    fn flow_restriction_factor(&self, current_pressure: Pressure) -> f64 {
+        // Selecting old 3000psi vs new hydraulic architecture based on max pressure of the system
+        // New 5000 psi systems have a higher flow degradation with pressure going lower
+        if self.max_working_pressure < Pressure::new::<psi>(4000.) {
+            (1. / (self.max_working_pressure.get::<psi>()
+                - Self::FLOW_REDUCTION_THRESHOLD_BELOW_MAX_PRESS_PSI)
+                * current_pressure.get::<psi>().powi(2)
+                * 1.
+                / (self.max_working_pressure.get::<psi>()
+                    - Self::FLOW_REDUCTION_THRESHOLD_BELOW_MAX_PRESS_PSI))
+                .min(1.)
```

Comment:
```suggestion
            (1. / (self.max_working_pressure.get::<psi>()
                - Self::FLOW_REDUCTION_THRESHOLD_BELOW_MAX_PRESS_PSI)
                * current_pressure.get::<psi>().powi(2)
                * 1.
                / (self.max_working_pressure.get::<psi>()
                    - Self::FLOW_REDUCTION_THRESHOLD_BELOW_MAX_PRESS_PSI))
                .min(1.)
```
This looks weird. You are dividing twice by `max_working_pressure * FLOW_REDUCTION_THRESHOLD_BELOW_MAX_PRESS_PSI` is this intended?